src/pages/Models/Details/__snapshots__/ModelDetails.test.js.snap

```diff
@@ -582,13 +582,16 @@ exports[`ModelDetail Container renders the details pane 1`] = `
         >
           <thead>
             <TableRow>
-              <tr>
+              <tr
+                role="row"
```

Comment:
It looks like the table in the repository doesn't contain the role removal updates

Reply:
I didn't remove them because the docs suggest they're required, it's possible that'll change in the next release, in which case we can move them then.

"Using the class p-table--sortable and assigning role="columnheader" and aria-sort to each <th> element will show each table column to be sortable..."

https://docs.vanillaframework.io/base/tables/#sortable